dhall/tests/format/stringWithNullB.dhall

```diff
@@ -0,0 +1,3 @@
+''
+${"\u0000"} $ \ 
+ ☺''
```

Comment:
It's a bit funny that this triggers #1545 now! ;)

Should I add the fix for this to this PR or should I keep it separate?

Reply:
See #1550.